### PR TITLE
42 test remove rpc

### PR DESCRIPTION
**Pull request by fmhoeger**
* Add tests for smaug wallet removal via RPC
* Make sure the smaug db file also gets deleted and add a corresponding test

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -374,8 +374,9 @@ async fn add(plugin: Plugin<State>, args: AddArgs) -> Result<serde_json::Value,
         .map_err(|e| anyhow!("Error calling listdatastore: {:?}", e))?;
     let name = &dw.get_name()?;
     let message = format!("Wallet with deterministic name {} successfully added", name);
+    let db_path = dw_clone.get_db_file_path(db_dir).unwrap();
     log::info!("{}", message);
-    Ok(json!({"name": name, "message": message}))
+    Ok(json!({"name": name, "message": message, "db_path": db_path}))
```
Comment (chrisguida) [inline src/main.rs:379]: Why are we outputting this to the user?

**Review by chrisguida — CHANGES_REQUESTED** [review]

**Comment by chrisguida** [comment]
Everything looks good except for the issue in the comment above

File: src/main.rs (hunk 1 above)
Comment (fmhoeger) [reply src/main.rs:379]: Currently, the returned `db_path` from the `add` RPC is used in the `remove` RPC test in order to be able to check for the existence of the smaug db file: `assert os.path.isfile(db_path)` and `assert not os.path.isfile(db_path)`, resp.
But yes, I am aware it is a bad idea to expose the db path in this way just to be able to accomplish testing. What would be a better way to accomplish this?

Comment (chrisguida) [reply src/main.rs:379]: Just calculate the path the same way we calculate it in smaug.

Should just be $LIGHTNING_DIR/.smaug/<wallet-name>.db

Comment (fmhoeger) [reply src/main.rs]: Fixed.

**Review by chrisguida — CHANGES_REQUESTED** [review]
LGTM

**Review by chrisguida — APPROVED** [review]

**Pull request merged**